tests/devices/qubit/test_simulate.py

```diff
@@ -1561,12 +1559,12 @@ def test_tree_traversal_combine_measurements(self, measurements, expected):
         else:
             assert qml.math.allclose(combined_measurement, expected)
 
-    @flaky(max_runs=3, min_passes=1)
+    @pytest.mark.local_salt(2)
```

Comment:
Each time we update the global salt, some tests may fail if it unfortunately receives a bad seed. If we modify the seed for these tests locally in the test function every time this happens, we can imagine that the number of tests with locally modified seeds will increase. The marker is my answer to a more standardized way of getting a new seed in case the original one doesn't work. In the future we can easily do a sweep of the entire test suite to get rid of these temporary local salts when possible.